pages/json.md

```diff
@@ -0,0 +1,52 @@
+#JSON
+
+##JSON implementation in Rust:
+
```

Comment:
Would you mind including a sentence or two about what this recipe demonstrates, please include links to the primary sources of documentation.